src/rules/latin-var-name-only-rule/closest-latin-to-cyrrilic-char-similarity.js

```diff
@@ -0,0 +1,32 @@
+let cyrrillicToLatinaChar = {
+	'А': 'A',
+	'а': 'a',
+	'В': 'B',
+	'в': 'b',
+	'С': 'C',
+	'с': 'c',
+	'Е': 'E',
+	'е': 'e',
+	'Н': 'H',
+	'н': 'h',
+	'К': 'K',
+	'к': 'k',
+	'М': 'M',
+	'м': 'm',
+	'О': 'O',
+	'о': 'o',
+	'Р': 'P',
+	'р': 'p',
+	'Т': 'T',
+	'т': 't',
+	'Х': 'X',
+	'х': 'x',
+	'У': 'Y',
+	'у': 'y'
+};
+
+function closestLatinToCyrrilicCharSimilarity(char){
```

Comment:
Functions names better to describe an action or a verb. This name is absolutely a noun